src/components/dialogs/PreferencesDialog/index.js

```diff
@@ -78,4 +78,13 @@ class PreferencesDialog extends Component {
   }
 }
 
+function parseAsNumberIfNumber (str) {
```

Comment:
Huh, I just realized the commit I'm replacing (preferences dialog) removes the only reference to lodash (the times for SpeedTabPanel was generated using lodash).

I think it'd be best to keep deps at the minimum, I'll remove the lodash dependency and I'll extract this helper function to util